hoot-core/src/main/cpp/hoot/core/io/OsmXmlReader.cpp

```diff
@@ -721,8 +721,8 @@ void OsmXmlReader::_uncompressInput()
 
     // "man gzip" confirms success return code is zero
     //  -d option is "decompress"
-    //  -k option is "keep," meaning don't delete input .osm.gz
-    const std::string cmd(std::string("gzip -d -k ") + originalFile.toStdString());
+    //  -c option is "write on standard output, keep original files unchanged" meaning won't delete input .osm.gz
+    const std::string cmd(std::string("gzip -dc ") + originalFile.toStdString() + " > " + _path);
```

Comment:
The build is failing on this, I think it needs a string object: `std::string(" > ")`.